integration-cli/docker_cli_network_unix_test.go

```diff
@@ -201,7 +201,8 @@ func isNwPresent(c *check.C, name string) bool {
 	out, _ := dockerCmd(c, "network", "ls")
 	lines := strings.Split(out, "\n")
 	for i := 1; i < len(lines)-1; i++ {
```

Comment:
I don't think we can use a `range` instead, because the first line (header line) and last line (trailing empty line) need to be discarded. note the for loop start and end:
```
for i := 1; i < len(lines)-1; i++ {
```
and I think `strings.Fields` will handle all space and `\t` things for us, so it's not necessary to trim space ourselves.
@cpuguy83